src/rendertron.ts

```diff
@@ -8,13 +8,15 @@ import * as koaLogger from 'koa-logger';
 import * as path from 'path';
 import * as puppeteer from 'puppeteer';
 import * as url from 'url';
+import { isArray, union } from 'lodash';
```

Comment:
Can we avoid dragging in this dependency for just these two simple features?

`isArray` -> `Array.isArray`
`union` -> `[...new Set([...a, ...b])];`

Reply:
@japboy Can I create a (new) PR to improve this? Or do you need to have some time and want to do it yourself?

Reply:
@japboy i'd be happy to help too

Reply:
Thanks for your patience, I will merge this PR and make the necessary changes.